requirements.txt

```diff
@@ -1,5 +1,6 @@
 numpy
 appdirs
-networkx
+networkx>=2.0
+matplotlib>=1.3.0
 clldutils>=1.6
-tqdm
+tqdm
```

Comment:
I think that "six" is missing.

Reply:
Ahh didn't spot that as it's installed by one of the prerequisites too (mock, I think)